File: TestIntern/Ex6.cpp
#include <iostream>
using namespace std;

    float Calculate(float x){
        float remaining_water, water_per_day;

        for (int i = 1; i <= 7; i++){
            water_per_day = x * (1.0 / 3);
            x = x - water_per_day;
            cout << " Remaining Water Day " << i << "  " << x << "Litre" <<endl;
        }

        remaining_water = x;
        return remaining_water;

    }
int main(){

    cout <<" Water in Tank  =  5832 Liter \n";
    float remaining_water = Calculate(5832);
    cout << "Water Remaining in 1 week " << remaining_water <<" Liter " << endl;
    return 0;
    
}

File: TestIntern/Ex6.java
public class Ex6 {

    public static float calculate(float x) {
        float remainingWater, waterPerDay;

        for (int i = 1; i <= 7; i++) {
            waterPerDay = x * (1.0f / 3);
            x = x - waterPerDay;
            System.out.println("Remaining Water Day " + i + "  " + x + " Litre");
        }

        remainingWater = x;
        return remainingWater;
    }

    public static void main(String[] args) {
        System.out.println("Water in Tank  =  5832 Liter");
        float remainingWater = calculate(5832);
        System.out.println("Water Remaining in 1 week " + remainingWater + " Liter");
    }
}